modules/client/src/main/java/org/apache/ignite/internal/client/sql/ClientStatement.java

```diff
@@ -31,14 +31,20 @@ public class ClientStatement implements Statement {
     private final String query;
 
     /** Default schema. */
+    @Nullable
```

Comment:
according to our [coding guidelines](https://google.github.io/styleguide/javaguide.html#s4.8.5-annotations), type-use annotations appear immediately before the annotated type. Please fix it in other places too

Reply:
Fixed.

I've used `javax.annotation.Nullable` instead of `org.jetbrains.annotations.Nullable` by mistake, and the IDE displays a warning about incorrect order in that case.